test_local.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -e
 
-black .
-python lint.py
+black . --line-length=100
```

Comment:
Technically this is already defined [here](https://github.com/flexcompute/tidy3d/blob/b6427aa9230192ba64ff35f769b8a7f989a7fa76/pyproject.toml#L8-L9) but I guess it doesn't hurt